index.html

```diff
@@ -13471,6 +13535,7 @@ <h2>Substantive changes targetted for the 1.3 release</h2>
 		<h2>Substantive changes since the last public working draft</h2>
 		<ul>
 			<!-- EdNote: After each WD publish, move contents of this list into the next one below. -->
+			<li>03-Feb-2020: Add <pref>aria-repeated</pref>.</li>
```

Comment:
Thanks for adding the ChangeLog entry! Based on experience, this tends to result in merge conflicts because things don't land right away and/or in the expected order. Therefore, James and I typically do the ChangeLog after. (I'm not saying don't include it; merely giving you the option.)